frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -17,11 +17,12 @@ import styles from './styles.module.scss';
 type Properties<T extends FieldValues> = {
     control: Control<T, null>;
     errors: FieldErrors<T>;
-    label: string;
+    label?: string;
     placeholder?: string;
     name: FieldPath<T>;
     type?: InputType;
-    className?: string;
+    input_className?: string;
+    label_className?: string;
```

Comment:
```suggestion
    labelClassName?: string;
```

Reply:
Resolved in [21ff6e3](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/142/commits/21ff6e343c8c4dca0ca77bf01a3f766999e3e475)